.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+indent_size = 2
+indent_style = space
+end_of_line = crlf

```

Comment:
@vinibeloni Why did you change it to `crlf`?

Reply:
haha oh, my mistake, because I committed changes before run eslint, thanks for letting me know